thunder/core/trace.py

```diff
@@ -188,6 +189,9 @@ def _make_name(self, *, prefix: str | None = None, is_object_name: bool = False,
                 self.obj_name_ctr += 1
                 if obj is None:
                     name = f"_object_{name}"
+                elif isinstance(obj, Enum):
+                    # even though it should be unique, we need to do the counter suffix for technical problems
```

Comment:
What technical problems? Should this link an issue?

Reply:
Yeah, once I get a traceback.